lib/ansible/modules/storage/netapp/na_ontap_disks.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/python
+
+# (c) 2018, NetApp, Inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+
+module: na_ontap_disks
+
+short_description: NetApp ONTAP Assign and unassign disks from nodes
+extends_documentation_fragment:
+    - netapp.na_ontap
+version_added: '2.7'
+author: NetApp Ansible Team (ng-ansibleteam@netapp.com)
+
+description:
+- Assign all or part of disks to nodes.
+
+options:
+
+  node:
+    required: true
+    description:
+    - It specifies the node to assign all visible unowned disks.
+
+'''
+
+EXAMPLES = """
+- name: Assign unowned disks
```

Comment:
How would you unassign, set `node:` to empty, null? 
Would be worth adding an example for that. 

If it's not possible then I think `short_description` may need updating